docs/reference/contributing/connectivity/CellularInterface.md

```diff
@@ -118,38 +89,50 @@ Setup for some predefined targets is available in [CellularTargets.h](https://os
 
 If none of the existing modems is compatible with the new modem, then create new modem implementation under [features/cellular/framework/targets](https://github.com/ARMmbed/mbed-os/blob/master/features/cellular/framework/targets). A device class inheriting [AT_CellularDevice](https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/_a_t___cellular_device_8h_source.html) is a minimum. [QUECTEL_BG96](https://github.com/ARMmbed/mbed-os/blob/master/features/cellular/framework/targets/QUECTEL/BG96/QUECTEL_BG96.h) is an example of the device class. You can create additional overlapping functionality for everything that is available in [features/cellular/framework/targets](https://github.com/ARMmbed/mbed-os/blob/master/features/cellular/framework/targets).
```

Comment:
We also don't link to GitHub if at all possible. Could you please replace GitHub links with Doxygen links? Let me know if I need to generate Doxygen for another feature branch here.

Reply:
Fixed CellularDevice and LISA PPP link https://github.com/AriParkkila/Handbook/commit/bb3b257e26fcdfc7edc4d9dff6fc94c07b5421fc

Example links are still to Github, and there is no reason to make Doxygen to those individual targets.

I will try to remove links in the next phase, as commented above.